samcli/commands/validate/validate.py

```diff
@@ -95,3 +108,63 @@ def _read_sam_file(template):
         sam_template = yaml_parse(sam_template.read())
 
     return sam_template
+
+
+def _lint(ctx: Context, template: str) -> None:
+    """
+    Parses provided SAM template and maps errors from CloudFormation template back to SAM template.
+
+    Cfn-lint loggers are added to the SAM cli logging hierarchy which at the root logger
+    configures with INFO level logging and a different formatting. This exposes and duplicates
+    some cfn-lint logs that are not typically shown to customers. Explicitly setting the level to
+    WARNING and propagate to be False remediates these issues.
+
+    Parameters
+    -----------
+    ctx
+        Click context object
+    template
+        Path to the template file
+
+    """
+
+    import cfnlint.core  # type: ignore
+    import logging
+    from samcli.commands.exceptions import UserException
+    from botocore.utils import validate_region_name
+    from botocore.exceptions import InvalidRegionError
+
+    LOGGER = logging.getLogger("cfnlint")
```

Comment:
Can we add some comments why we need these logging settings? And also change logger name to something that will hint this is related to cfn-lint? Lastly, I don’t think this should be all uppercase since it is not constant anymore, it is just another variable in a function.

Reply:
Will change the name. As for comments, I had added some above the logger but pylint was reporting some formatting errors. The comments are located right below the _lint function now. Let me know if those are sufficient.